src/Datadog.Trace.ClrProfiler.Native/integration_loader.h

```diff
@@ -17,17 +17,31 @@ using json = nlohmann::json;
 
 // LoadIntegrationsFromEnvironment loads integrations from any files specified
 // in the DD_INTEGRATIONS environment variable
-std::vector<Integration> LoadIntegrationsFromEnvironment();
+void LoadIntegrationsFromEnvironment(std::vector<IntegrationMethod>& integrationMethods, const bool isCallTargetEnabled,
+                                     const bool isNetstandardEnabled,
+                                     const std::vector<WSTRING> disabledIntegrationNames);
+
 // LoadIntegrationsFromFile loads the integrations from a file
-std::vector<Integration> LoadIntegrationsFromFile(const WSTRING& file_path);
+void LoadIntegrationsFromFile(const WSTRING& file_path, std::vector<IntegrationMethod>& integrationMethods,
+                              const bool isCallTargetEnabled, const bool isNetstandardEnabled,
+                              const std::vector<WSTRING> disabledIntegrationNames);
+
 // LoadIntegrationsFromFile loads the integrations from a stream
-std::vector<Integration> LoadIntegrationsFromStream(std::istream& stream);
+void LoadIntegrationsFromStream(std::istream& stream, std::vector<IntegrationMethod>& integrationMethods,
+                                const bool isCallTargetEnabled, const bool isNetstandardEnabled,
+                                const std::vector<WSTRING> disabledIntegrationNames);
 
 namespace
 {
 
-    std::pair<Integration, bool> IntegrationFromJson(const json::value_type& src);
-    std::pair<MethodReplacement, bool> MethodReplacementFromJson(const json::value_type& src);
+    void IntegrationFromJson(const json::value_type& src, std::vector<IntegrationMethod>& integrationMethods,
+                             const bool isCallTargetEnabled, const bool isNetstandardEnabled,
+                             const std::vector<WSTRING> disabledIntegrationNames);
+
+    void MethodReplacementFromJson(const json::value_type& src, const WSTRING integrationName,
```

Comment:
you can pass `instegrationName` by `const &` instead

Reply:
done in 1aa6b9d